### PR TITLE
fix(EA-2444): Changed from node_env to NODE_ENV.

### PR DESCRIPTION
**Pull request by brunohadlich**
- CCW uses uppercase for env variables so I fixed the variable this repo reads from `node_env` to `NODE_ENV`.
- Check if project is configured to run E2E tests, if it doesn't and NODE_ENV === 'prod' then we close the milestone.

### CODE REVIEW

**Review by ThePinzon — DISMISSED** [review]

File: src/run/run.ts [hunk 1]
```diff
@@ -24,12 +24,33 @@ export async function run(): Promise<void> {
...
+        // get the milestone created in the build/test-unit job
+        const milestone = await testrailService.establishMilestone().catch((error) => {
+          setFailed("A TestRail Milestone could not be established.");
+          throw error;
+        });
+        // and close it
+        await testrailService.closeMilestone(milestone.id).catch((error) => {
+          setFailed("The TestRail Milestone could not be closed.");
+          throw error;
+        });
```
Comment (vitalizatroutine) [inline src/run/run.ts:46]: I think it would be a lot cleaner to separate this into its own function and give it a good name.

Comment (brunohadlich) [reply src/run/run.ts]: Done in https://github.com/q4mobile/stratus-testrail-reporter/pull/33/commits/42ff252d20ab132dc2b40690c3c5fc8094623467.

File: src/run/run.ts [hunk 2]
```diff
@@ -2,10 +2,36 @@ import { getBooleanInput, getInput, setFailed, setOutput } from "@actions/core";
 import { INewTestRun } from "testrail-api";
 import { extractError } from "../utils";
 import { Environment, InputKey, RunInputs, TestRunConfig, TestRailOptions } from "./run.definition";
-import { extractTestResults, getTrunkTestRunConfigs } from "./run.utils";
+import { containsE2Etest, extractTestResults, getTrunkTestRunConfigs, getUnitTestConfig } from "./run.utils";
 import { TestrailService } from "../services";
 import findDuplicates from "../utils/findDuplicate";
 
+async function closeMilestoneWithOnlyUnitTest(
```
Comment (vitalizatroutine) [inline src/run/run.ts:9]: i would move this function below `run`, as `run` is the primary function here and should be read first. Perhaps right above or below `reportToTestrail`.

Comment (brunohadlich) [reply src/run/run.ts]: Done in https://github.com/q4mobile/stratus-testrail-reporter/pull/33/commits/0ef07600b532e8ef013028d78fe9c0b4e574f0cb.

**Review by vitalizatroutine — DISMISSED** [review]

**Review by ThePinzon — APPROVED** [review]

**Pull request merged**